### DISCUSSION TITLE
The End of Crashing Waves

### DISCUSSION THREAD

**Discussion by Raytwo** [Raytwo/ARCropolis#540] [Announcements]
### Regression(s)
* Due to changes in a newer Rust version, some MSBT patches might not work properly anymore (which is why this version is 4.0.0)

### Bugfixe(s)
* Improve support for directory addition (@itsmeft24)
* Keep MSBT support working as best as possible (@ThatNintendoNerd)

# Read this
Skyline is not shipped with ARCropolis anymore.
Please read [this issue](https://github.com/Raytwo/ARCropolis/issues/157) to know how to proceed from now on.

<hr /><em>This discussion was created from the release <a href='https://github.com/Raytwo/ARCropolis/releases/tag/v4.0.0'>The End of Crashing Waves</a>.</em>

**Comment by MikeyProHD** [comment]
One problem I've noticed with multiple Arcropolis versions (including this beta one) is that when you have a lotta mods installed with added slots in regular smash mode, the game runs fine on the CSS with it only loading whatever slot the cursor is hovering over. I'm like 99.9% sure this isn't the case for Squad Strike, though. When I go into the CSS for Squad Strike the game Crashes within the first 10-15 seconds. I've also had rare times where even in the regular smash CSS the game would crash, but that's pretty rare.

**Reply by Raytwo** [reply]
I think that's because the CSS are not the same and the lazy loading doesn't apply there, might want to ask @Coolsonickirby to look into it

**Reply by MikeyProHD** [reply]
Oh ok I'll do that, thx!

**Reply by youdu001** [reply]
4.0.2
Error Code:2-0069
ARCropolis has panicked! Please open the detailsand send a screenshot to the developer. then closethe game.

**Comment by SaxxyBoy328** [comment]
Hey so I got this crash message after installing a couple mods. Any idea as to what this could be?
![arcropolis crash image](https://github.com/Raytwo/ARCropolis/assets/144612021/9fab9639-0d1f-4b99-a6e1-9e2d27519361)

**Reply by SaxxyBoy328** [reply]
I'm thinking it had something to do with this mod because I had to replace the 'ultimate' folder in the sd card directory:

https://gamebanana.com/mods/456265

**Comment by Ploots** [comment]
Hey, thanks for the 4.0.0 version! Apparently this is the only version that seems to work on my switch at all. I was having so many problems before from a (regrettably outdated) youtube video installing mods and updating to this version seemed to do the trick! Thanks!

**Comment by ToasterClock202** [comment]
My game just doesnt load like not even black sceen it just leaves me at the ryujinx menu and this string of code caught my eye

00:00:08.882 |I| HLE.OsThread.43 ServiceSfdnsres GetAddrInfoRequestImpl: Trying to resolve: api.github.com

idk if that will help but thats all i got

**Comment by KaminaMamoru** [comment]
Hey, smash goes on a black screen then crashes whenever you go on QP or make/join an arena since the new 13.0.2 smash update. Any fix for this? You can't play online anymore rn.

**Reply by Skysever** [reply]
I found out removing one slot effect (.nro) plugin located in atmosphere folder was the culprit for me. It may help you too

**Comment by Sushi375** [comment]
With the most recent ARCropolis update, when I load in to smash bros, my screen is stuck on the black screen with nintendo logos (see picture below). Suggestions on how to fix this would be appreciated.
![IMG_1402](https://github.com/Raytwo/ARCropolis/assets/160216149/6fe00f5a-d234-40fd-86a5-ffc06b659660)

**Reply by userbouy** [reply]
Try updating Smash to 13.0.2

**Comment by Xalavar** [comment]
The latest update results in a black screen whenever trying to load replays that were saved during Update 13.0.1 and earlier.

**Comment by userbouy** [comment]
When will android support drop? I mean, I BADLY wanna play HDR on my phone. You know, constantly using my laptop isn't really comfortable, so PLEASE, I BEG YOU FOR ANDROID SUPPORT🙏🙏🙏🙏
